astropy/nddata/utils.py

```diff
@@ -861,23 +842,34 @@ def origin_cutout(self):
         """
         return (self.slices_cutout[1].start, self.slices_cutout[0].start)
 
+    @staticmethod
+    def _round(a):
+        """
+        Round the input to the nearest integer.
+
+        If two integers are equally close, the value is rounded up.
+        Note that in this case `np.round` rounds to the nearest even
```

Comment:
Maybe 'Note that this is different from `np.round` which ...

Reply:
Just out of curiosity, why make this a static method?

Reply:
It's just a generic rounding function that I use for several different inputs.  I'm happy to change it to a regular method, but it doesn't use anything defined in the class instance (i.e. `self`).  I thought in that case that a static method was appropriate.  Which do you prefer?

Reply:
Sorry I meant as opposed to just being a function

Reply:
I made it a static method because it's now only being used by `Cutout2D`.  Previously it was also being used by `overlap_slices` (and thus a private function).  I'm inclined to keep it a static method unless you feel strongly about it.